app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class API::UsersController < API::RestfulController
+  def index
```

Comment:
I don't think you want to make a new controller here... we have the ProfileController which acts as our UsersController at this point, which `Records.users` points to... easier to stick with that I believe.

Reply:
Then we don't have to do all these gymnastics around moving apiEndpoint around; if anything we should consider renaming ProfileController to UsersController, which is what it always wanted to be anyway.

Reply:
Adding `profile#index` instead of `users#index` cuts this PR down to about 10 files, instead of 21

Reply:
ok thanks will do. I think the move of apiendpoint is technically right, but agree it does not need to be included here.